controllers/application/deployment.go

```diff
@@ -88,9 +89,13 @@ func (r *ApplicationReconciler) defineDeployment(ctx context.Context, applicatio
 		podTemplateLabels = util.GetPodAppSelector(application.Name)
 	}
 
+	// Add annotations to pod template, safe-to-evict added due to issues
+	// with cluster-autoscaler and unable to evict pods with local volumes
+	// https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md
 	generatedSpecAnnotations := map[string]string{
-		"argocd.argoproj.io/sync-options": "Prune=false",
-		"prometheus.io/scrape":            "true",
+		"argocd.argoproj.io/sync-options":                "Prune=false",
+		"prometheus.io/scrape":                           "true",
+		"cluster-autoscaler.kubernetes.io/safe-to-evict": "true",
```

Comment:
maybe add this to the minimal test, to confirm its been set

Reply:
Added test in https://github.com/kartverket/skiperator/pull/396/commits/f34ba24781cff6adec18506520362118a9e35ef1